source/StigData/Processed/VMW-vSphere-7-0-1.2.xml

```diff
@@ -0,0 +1,1352 @@
+
+Verify that only the Administrator, No Trusted Infrastructure Administrator, vCLSAdmin, and vSphere Kubernetes Manager and any site-specific cryptographic roles have the following permissions:
+
+Cryptographic Operations privileges
+Global.Diagnostics
+Host.Inventory.Add host to cluster
+Host.Inventory.Add standalone host
+Host.Local operations.Manage user groups
+
+or
+
+From a PowerCLI command prompt while connected to the vCenter server, run the following commands:
+
+$roles = Get-VIRole
+ForEach($role in $roles){
+    $privileges = $role.PrivilegeList
+    If($privileges -match "Crypto*" -or $privileges -match "Global.Diagnostics" -or $privileges -match "Host.Inventory.Add*" -or $privileges -match "Host.Local operations.Manage user groups"){
+    Write-Host "$role has Cryptographic privileges"
+    }
+}
+
+If any role other than the four default roles contain the permissions listed above and is not authorized to perform cryptographic-related operations, this is a finding.</RawString>
+    </Rule>
+  </PermissionRule>
+  <VsphereAdvancedSettingsRule dscresourcemodule="Vmware.vSphereDSC">
+    <Rule id="V-256321" severity="medium" conversionstatus="pass" title="SRG-APP-000095" dscresource="VMHostAdvancedSettings">
+      <AdvancedSettings>'' = 'info'</AdvancedSettings>
+      <Description>&lt;VulnDiscussion&gt;Without establishing what types of events occurred, it would be difficult to establish, correlate, and investigate the events leading up to an outage or attack.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
+      <DuplicateOf />
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>From the vSphere Client, go to Host and Clusters.
+
+Select a vCenter Server &gt;&gt; Configure &gt;&gt; Settings &gt;&gt; Advanced Settings.
+
+Verify the "config.log.level" value is set to "info".
+
+or
+
+From a PowerCLI command prompt while connected to the vCenter server, run the following command:
+
+Get-AdvancedSetting -Entity &lt;vcenter server name&gt; -Name config.log.level and verify it is set to "info".
+
+If the "config.log.level" value is not set to "info" or does not exist, this is a finding.</RawString>
+    </Rule>
+    <Rule id="V-256355" severity="medium" conversionstatus="pass" title="SRG-APP-000516" dscresource="VMHostAdvancedSettings">
+      <AdvancedSettings>'' = '30'</AdvancedSettings>
```

Comment:
this setting is not correct, should be 
'VirtualCenter.VimPasswordExpirationInDays' = '30'

Reply:
Corrections Made

Reply:
@erjenkin  Can you help us with the rules that are failing to convert. Also at the moment, only three rules are automated.